ui/app/models/sync/destinations/vercel-project.js

```diff
@@ -3,30 +3,41 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
+import { isArray } from '@ember/array';
```

Comment:
I don't think this has an advantage over `Array.isArray` for this use case and then you wouldn't need the import.